src/Discord/Parts/User/User.php

```diff
@@ -204,6 +207,45 @@ protected function getAvatarHashAttribute(): ?string
         return $this->attributes['avatar'];
     }
 
+    /**
+     * Returns the avatar decoration URL for the client.
+     *
+     * @param string|null $format The image format.
+     * @param int         $size   The size of the image.
+     *
+     * @return string|null The URL to the clients avatar decoration.
+     */
+    public function getAvatarDecorationAttribute(?string $format = null, int $size = 288): ?string
+    {
+        if (! isset($this->attributes['avatar_decoration'])) {
+            return null;
+        }
+
+        if (isset($format)) {
+            $allowed = ['png', 'jpg', 'webp'];
+
+            if (! in_array(strtolower($format), $allowed)) {
+                $format = 'png';
+            }
+        } elseif (strpos($this->attributes['avatar_decoration'], 'a_') === 0) {
```

Comment:
Looks like the default is `png` whether its animated or not. Wouldn't it make more sense to change the input to `string $format = 'png'` rather than have these checks?